Rakefile

```diff
@@ -8,7 +8,7 @@ require "ruby_lsp/check_docs"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb"]
+  t.test_files = FileList["test/**/*_test.rb"].exclude("test/fixtures/**/*.rb")
```

Comment:
I don't think this change is necessary since all fixtures are `txt` files.

Reply:
Oh, right. I didn't think about YARP's own tests. Do you know if there's some git submodule magic that will allow us to only pull the `txt` fixtures?

If not, that's fine, let's move forward with the exclude.

Reply:
As far as I know, this is not possible with a submodule command.

What may be possible is to delete all files except fixtures in the [test](https://github.com/Shopify/ruby-lsp/blob/main/bin/test) script. But this is a kind of "workaround" 😄